cgra/CgraRTL.py

```diff
@@ -105,13 +105,17 @@ def construct(s, DataType, PredicateType, CtrlPktType, CtrlSignalType,
     s.recv_from_cpu_pkt //= s.controller.recv_from_cpu_pkt
 
     # Connects ring with each control memory.
-    for i in range(s.num_tiles):
-      s.ctrl_ring.send[i] //= s.tile[i].recv_ctrl_pkt
+    s.ctrl_ring.send[0] //= s.controller.recv_from_ctrl_ring_ctrl_pkt

```

Comment:
Rename `controller.recv_from_ctrl_ring_ctrl_pkt` -> `controller.recv_from_ctrl_ring_pkt`?

And can we connect it to `s.ctrl_ring.send[s.num_tiles]` rather than `0`?